lease/lessor.go

```diff
@@ -26,6 +26,15 @@ var (
 	minLeaseTerm = 5 * time.Second
 )
 
+// DeleteableKV defines a KV interface with DeleteRange method.
+// We define this interface only for lessor to limit the number
+// of methods of storage.KV to what lessor actually needs.
+//
+// Having a minimum interface makes testing easy.
+type DeleteableKV interface {
```

Comment:
The name is not good IMO. For DeleteableKV, i would expect it to be a full KV with deletion functionality. If it is just for fast move, i am ok with that.